package.json

```diff
@@ -242,6 +242,7 @@
     "eslint-plugin-import": "2.8.0",
     "eslint-plugin-jest": "^21.6.2",
     "eslint-plugin-mocha": "4.11.0",
+    "eslint-plugin-no-unsanitized": "^2.0.2",
```

Comment:
This should also be added as a `peerDependencies` in `packages/eslint-config-kibana/package.json`

Reply:
@kjbekkelund Any clue why we don't just add all of these eslint dependencies as regular dependencies to `packages/eslint-config-kibana` rather than requiring peerDependencies?

Reply:
https://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config

> You should declare your dependency on ESLint in package.json using the peerDependencies field. 

Maybe we don't need it anymore. And maybe @spalger knows more details here.

Reply:
Added to peerDependencies for now. Though we should figure that one out separately because it's a pretty silly workflow.

Reply:
++ would be nice to get away from it. And we might not need to do this at all because this isn't _really_ a _published_ config anymore. It might just be a recommendation that fits "the larger community", not necessarily our context.